### PR TITLE
feat(aureliaCLICommands): add au run to command pallet

### PR DESCRIPTION
**Pull request by SteveHartzog**
Add au run command to the command pallet. Did I miss anything or is that it?

### CODE REVIEW

File: src/client/aureliaCLICommands.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export default class AureliaCliCommands {
       vscode.commands.registerCommand('extension.auGenerate', () => this.auGenerate(outputChannel)),
       vscode.commands.registerCommand('extension.auTest', () => this.runCommand(['test'], outputChannel, false)),
       vscode.commands.registerCommand('extension.auBuild', () => this.auBuild(outputChannel)));
```
Comment (eriklieben) [inline src/client/aureliaCLICommands.ts]: Thanks for the pull request!

If you add the line above the auBuild line or change the above line to end on ```(outputChannel)),``` and the new line on ```outputChannel, true)));``` then the build will succeed.

You could test if the build succeeds by running ```npm run build``` on your local machine, once you done a ```npm install```.

Then we can take a look and test it :-)

Comment (SteveHartzog) [inline src/client/aureliaCLICommands.ts]: Oops, my bad. I didn't see that I had added a trailing comma after the statement was completed; and sadly I didn't run the build after making the changes. I did run the tests however, but of course no tests hit this file. This is my first pull request anywhere. Hopefully my next one will be better. :-)

Comment (SteveHartzog) [inline src/client/aureliaCLICommands.ts]: I looked over the helloworld vscode extension example/docs and noticed I had forgot to expose the command in the package.json. I've tested with F5 and it's there now.

Comment (SteveHartzog) [inline src/client/aureliaCLICommands.ts]: I just looked over my code after going through the hello world vscode extension sample and noticed that I needed to update the package.json to expose the new command... in two places. Tested and confirmed to run auRun >> au run.

**Comment by eriklieben** [comment]
Tested it & merged it.

Thank you for helping out improving the visual studio code Aurelia experience!

**Comment by eriklieben** [comment]
Just released a new version 0.1.11, which includes your change, sorry for the delay.

Thank you for the contribution to making the Aurelia vs code editor experience great!

**Pull request merged**